index.js

```diff
@@ -136,7 +136,9 @@ function packAndInstallLibrary(name, dir, targetDir) {
     }
     fs.mkdirSync(libDestDir, { recursive: true })
 
-    const regex = new RegExp(`^${name}(.*).tgz$`)
+    const tmpName = name.replace(/[\s\/]/g, "-").replace(/@/g, "at-")
```

Comment:
"at-" ???  Is it right?

Reply:
I went with restoring the original logic here. Didn't look too much into the reasoning for when you'd need to have "at". My initial testing I found like you suggested on the diff where the packed filename seems to omit it by default.

https://github.com/mweststrate/relative-deps/commit/e23ff45128346cac0d74a0f1d09c649314008f22#diff-168726dbe96b3ce427e7fedce31bb0bcL138